purchase_request_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Operating Unit in Purchase Requests",
+    "version": "10.0.1.0.0",
+    "author": "Eficent",
+    "website": "http://www.eficent.com",
+    "category": "Purchase Management",
+    "depends": ["purchase_request",
+                "purchase_operating_unit"],
+    "description": """
+Operating Unit in Purchase Requests
```

Comment:
This is actually a summary. should be shorter. Also add the licence key:  `"license": "LGPL-3",`